media-tv/plex-media-server/plex-media-server-1.23.1.4571.ebuild

```
@@ -0,0 +1,80 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+MY_COMMIT="6119e8eed"
+MY_PV="${PV}-${MY_COMMIT}"
```

Let's take nitpicking this one further... seeing how it's only used once, maybe combine?
```suggestion
MY_PV="${PV}-6119e8eed"
```